MaterialLib/FractureModels/Permeability/CreateCubicLaw.cpp

```diff
@@ -0,0 +1,27 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "CreateCubicLaw.h"
+
+#include "BaseLib/ConfigTree.h"
+
+#include "CubicLaw.h"
+
+namespace MaterialLib::Fracture::Permeability
+{
+std::unique_ptr<Permeability> createCubicLaw(BaseLib::ConfigTree const& config)
+{
+    //! \ogs_file_param{prj__processes__process__HYDRO_MECHANICS_WITH_LIE__fracture_properties__permeability_model__type}
```

Comment:
The model is more general.  It is better to change the path to
`//! \ogs_file_param_special{material__porous_medium__fracture_properties__permeability_model__type}`.